server/public/landing/landing.controller.js

```diff
@@ -4,26 +4,31 @@
     .module('jubilant-umbrella.landing')
     .controller('LandingController', LandingController)
 
-  function LandingController ($state) {
+  function LandingController (Schools, $state, $http) {
```

Comment:
Your dependencies should be alphabetized, with third party modules coming after custom modules (switch $state and $http).